src/google/protobuf/json/json.h

```diff
@@ -69,6 +69,21 @@ inline absl::Status MessageToJsonString(const Message& message,
   return MessageToJsonString(message, output, PrintOptions());
 }
 
+// Converts from protobuf message to JSON and appends it to |json_output|. This is a
+// simple wrapper of BinaryToJsonString(). It will use the DescriptorPool of the
+// passed-in message to resolve Any types.
+//
+// Please note that non-OK statuses are not a stable output of this API and
+// subject to change without notice.
+PROTOBUF_EXPORT absl::Status MessageToJsonStream(const Message& message, 
```

Comment:
I think we want to go with just `MessageToJson` as the name of this function. Adding a `Stream` or `String` suffix is something we used to do because our style guide used to be against function overloads with different types, but now we are OK with that.

Reply:
This would seem to introduce API-breaking changes. . Is this ok?

Reply:
Or, instead of modifying the original API, add a new function named `MessageToJson`.

Reply:
done